src/test/java/persistence/sql/dml/builder/InsertQueryBuilderTest.java

```diff
@@ -25,10 +25,11 @@ class withValidArgs {
             @Test
             @DisplayName("적절한 statement를 반환한다.")
             void returnStatement() {
+                Person person = new Person(1L, "민준", 29, "민준.com");
                 EntityAttribute entityAttribute =
                         EntityAttribute.of(TestEntityFixture.SampleOneWithValidAnnotation.class, attributeParser);
                 InsertQueryBuilder insertQueryBuilder = new InsertQueryBuilder();
-                String statement = insertQueryBuilder.prepareStatement(entityAttribute.createEntityContext(person));
+                String statement = insertQueryBuilder.prepareStatement(entityAttribute, person);
                 assertThat(statement)
                         .isEqualTo("INSERT INTO entity_name( id, name, old ) VALUES ( '1', '민준', '29' )");
```

Comment:
```suggestion
Person person = new Person(1L, "민준", 29, "민준.com");

                EntityAttribute entityAttribute = EntityAttribute.of(Person.class, attributeParser);
                InsertQueryBuilder insertQueryBuilder = new InsertQueryBuilder();
                String statement = insertQueryBuilder.prepareStatement(entityAttribute, person);
                assertThat(statement)
                        .isEqualTo("INSERT INTO users( id, nick_name, old, email ) VALUES ( '1', '민준', '29', '민준.com' )");
```

민준님 testFixture 와 생성하신 Person 객체의 차이가 무엇일까요?
생성해 주신 Person 이 픽스쳐의 역할을 한다고 생각했는데 별도의 테스트 픽스처가 있는 이유를 잘 모르겠습니다🥲

Reply:
처음에 Person으로 사용하다가 비지니스 로직이 변경됨에따라 Person이 바뀔수도있을것같아서 변하지않는 픽스처로 고정시키려고 테스트 엔티티를 만들었는데, 코드가 약간 혼재된것같네요.

Person 은 제거하고 테스트 엔티티로 전부 변경할까하는데 어떨까요?

Reply:
아 예 Person 으로 통일되든 테스트 픽스처로 통일되는 선택이면 될 것 같습니다!